packages/bpk-component-infinite-scroll/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "bpk-component-infinite-scroll",
-  "version": "1.0.5",
+  "version": "1.0.6",
```

Comment:
The update on the package.json is not needed as lerna will do that at release time

Reply:
changed